packages/next/src/utils.ts

```diff
@@ -48,6 +48,9 @@ export const MIB = 1024 * KIB;
 
 export const prettyBytes = (n: number) => bytes(n, { unitSeparator: ' ' });
 
+export const RSC_CONTENT_TYPE = 'x-component';
```

Comment:
We should probably put this in the `routes-manifest` in Next.js and keep that as the source of truth in case we need to change it ever